daemon/cluster/cluster.go

```diff
@@ -626,10 +626,6 @@ func (c *Cluster) stopNode() error {
 		return nil
 	}
 	c.stop = true
-	if c.cancelDelay != nil {
-		c.cancelDelay()
-		c.cancelDelay = nil
-	}
```

Comment:
Trying to understand moving this to `Cleanup`. Doesn't the call to `stopNode` from `Init` still need it?